plugins/importtext_procedure.cc

```diff
@@ -77,7 +77,7 @@ ImportTextConfigDescription::ImportTextConfigDescription()
     addField("named", &ImportTextConfig::named,
              "Row name expression for output dataset. Note that each row "
              "must have a unique name.",
-             SqlExpression::parse("lineNumber()"));
+             SqlExpression::parse("dataFileUrl() + '.' + lineNumber()"));
```

Comment:
I'm not clear on your suggestion. My 2 attemps don't work.
* `dataFileUrl(), lineNumber()` fails with `dataFileUrl(), lineNumber():1:14 ('dataFileUrl()>>>,<<< lineNumber('): expected eof`
* `[dataFileUrl(), lineNumber()]` fails with `Cannot convert value of type 'embedding' to 'atomic value' (Utf8 string)`

Why isn't the way I did it ok?